apps/wfo-ui-surf/pages/_app.tsx

```diff
@@ -137,4 +112,12 @@ function CustomApp({ Component, pageProps }: AppProps) {
     );
 }
 
+CustomApp.getInitialProps = async (
```

Comment:
I do still wonder if this is not an over complication and if you can't just use this pattern for all values in the initialOrchestratorConig

```    environmentName:
        process.env.NEXT_PUBLIC_ENVIRONMENT_NAME ?? Environment.DEVELOPMENT```

Reply:
According to our test-engineer the current way is higly desired since he does not have to wait long between deploying test environments. We might want to try to make our container image smaller.